cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,78 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+const { faker } = require('@faker-js/faker');
+
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new homePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+    });
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
+    const userNewName = faker.person.firstName().toLowerCase();
```

Comment:
Convert the username to lowercase might not be necessary as most systems are case-insensitive. If the system under test is case-sensitive, ignore this comment.

Reply:
userName without  String.toLowerCase() throw Error.